slides/components/auth-page.ftd

```diff
@@ -1,4 +1,5 @@
 -- import: ui.fifthtry.com/slides/site-data
+-- import: ui.fifthtry.com/slides/user-data
```

Comment:
No, do not do this. Login page should not be visible at all if user is logged in.